ckanext/dgu/theme/templates/_dgu_jinja_util.html

```diff
@@ -117,6 +117,39 @@
               </button>
             </span>
           </div>
+
+          <div>
+      <span class="dropdown">
+          <a href="#" class="dropdown-button" data-placement="top" data-delay="300" data-toggle="dropdown" title="" data-original-title="Advanced Search" style="color: #666; font-size: 12px; text-decoration: underline; text-align:center; display: block;">
+            Search Tips
+          </a>
+
+          <div class="panel panel-default dropdown-menu" style="width: 500px; margin-left: -1px;">
+            <div class="panel-heading" style="margin-top: -10px;"><strong>Search Tips</strong></div>
+            <div class="panel-body">
+
+                <ul>
+                  <li><span class='search-terms'>"armed forces" expenses</span> - search for exact phrases by putting quote marks around them</li>
+                  <li><span class='search-terms'>primary school -care age</span> - exclude words or quoted phrases by using a minus sign or NOT</li>
+                  <li><span class='search-terms'>primary OR secondary school</span> - use 'OR' to allow alternatives (the default is to 'AND' all of the terms)</li>
+                  <li><span class='search-terms'>publisher:cabinet-office res_format:CSV</span> - search only in a particular field. A list of available fields is shown in the <a href="http://guidance.data.gov.uk/finding_data.html" target="_blank">Finding Data guidance</a></li>
+                </ul>
+
+                <p>For more detail of the syntax, consult the <a href="https://lucene.apache.org/core/4_0_0/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#minitoc-area" target="_blank">Lucene documentation</a></p>
```

Comment:
We have assembled a hierarchy of content - the pop-up has the most important stuff, the guidance has the next most important, and the lucene docs fills in the gaps for the real experts. I don't think the lucene docs should have the prominance of being on the pop-up.